IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -590,6 +610,84 @@ public static AnyBitmap CreateMultiFrameGif(IEnumerable<AnyBitmap> images)
             return AnyBitmap.FromStream(stream);
         }
 
+        /// <summary>
+        /// Gets the stride width (also called scan width) of the <see cref="AnyBitmap"/> object.
+        /// </summary>
+        public int Stride
+        {
+            get
+            {
+                return GetStride();
+            }
+        }
+
+        /// <summary>
+        /// Gets the address of the first pixel data in the <see cref="AnyBitmap"/>. This can also be thought of as the first scan line in the <see cref="AnyBitmap"/>.
+        /// </summary>
+        /// <returns>The address of the first 32bpp BGRA pixel data in the <see cref="AnyBitmap"/>.</returns>
+        public IntPtr Scan0
+        {
+            get
+            {
+                return GetFirstPixelData();
+            }
+        }
+
+        /// <summary>
+        /// The MIMEType must be one of the following: 
```

Comment:
Returns the http mime type of the image.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types